### ISSUE TITLE
open new instance in new session, horizontal, virtical split always in home directory

### ISSUE THREAD

**Issue by arshavindn** [gnunn1/terminix#238]
From nemo, "open in terminal" from right-click menu context that opens new instance in new session always in home directory and in horizontal, virtical split does not work. Btw, new window seems work fine.

**Comment by gnunn1** [comment]
Did Nemo used to work for you until the last release? This might be related to the change I had to make to fix a bug for the Ubuntu launcher that wasn't setting a working directory and instead relying on PWD, I probably need to to look at this more holistically or just copy what gnome-terminal does with the assumption that is the expected behavior.

As to the spit, I got over-exuberant last release by changing the action names for split-horizontal and split-vertical to split-right and split-down, I didn't do enough due diligence to make sure I got all the spots where it was referenced. I'll be fixing this tonight and likely rolling out a bug fix release.
